src/app/frontend/chrome/chrome.html

```diff
@@ -23,10 +23,7 @@
       <h2>
         <span>kubernetes</span>
       </h2>
-      <span flex></span>
-      <md-button class="md-icon-button">
-        <md-icon md-font-library="material-icons">more_vert</md-icon>
-      </md-button>
+      <div flex ui-view="toolbar"></div>
```

Comment:
I think it is best to create chrome_state.js file and `export const toolbarViewName = 'toolbar';` there. Then import it in places that use it. With that you have one magic string less and can do renaming and discovery quickly.